src/components/Avatar/Avatar.test-int.js

```diff
@@ -0,0 +1,104 @@
+/* global Feature, Scenario */
+const searchWithinIframe = require('../../../test/codecept/utils/searchWithinIframe.js');
+
+const KIND = 'Avatar.Avatar';
+
+Feature('Avatar');
+
+Scenario('Overview', (I) => {
+    searchWithinIframe(I, KIND, 'Overview', () => {
+        I.see('Avatar: Avatar');
+        I.see('Overview');
+        I.see('Example');
+        I.seeElement('.uir-avatar');
+        I.see('DS');
+        I.see('Source Code');
+        I.seeElement('code');
+        I.see('Props');
+        I.seeElement('table');
+        I.see('property');
+        I.see('value');
+        I.see('required');
+        I.see('default');
+        I.seeNumberOfElements('td', 12 * 4);
+    });
+});
+
+Scenario('Initials', (I) => {
+    searchWithinIframe(I, KIND, 'Initials', () => {
+        I.see('Initials');
+        I.see('Variations');
+        I.seeNumberOfElements('.uir-avatar', 3);
+    });
+});
+
+Scenario('Image', (I) => {
+    searchWithinIframe(I, KIND, 'Image', () => {
+        I.see('Image');
+        I.seeNumberOfElements('.uir-avatar', 5);
```

Comment:
Since we use the same class name for avatar-initials and avatar-images, the 'Initials' and 'Image' test scenarios seem to validate the same state (beside the avatar sizes, but those class names are again generic for avatar-initials and avatar-images).

Again, a discussion to be had over our scope of unit and integration tests

Reply:
Separately, I remember us saying that we can't verify avatar rendering at a unit test level, but we do have a test for displaying user profile image. Was that a mistaken assumption initially?